src/phoenix/server/api/types/PromptVersionTemplate.py

```diff
@@ -0,0 +1,89 @@
+# Part of the Phoenix PromptHub feature set
+
+from enum import Enum
+from typing import Any, Union
+
+import strawberry
+from pydantic import BaseModel
+from strawberry.scalars import JSON
+
+JSONSerializable = Union[None, bool, int, float, str, dict[str, Any], list[Any]]
+
+
+@strawberry.enum
+class PromptMessageRole(str, Enum):
+    USER = "user"
+    SYSTEM = "system"
+    AI = "ai"  # E.g. the assistant. Normalize to AI for consistency.
+
+
+class PromptStringTemplate(BaseModel):
```

Comment:
I see the appeal of keeping the Pydantic models next to their corresponding GraphQL types, but will these models also be needed for the REST API?

Reply:
Yeah they will - let's de-couple

Reply:
@axiomofjoy I will say decoupling these into different files might lead to some weird circular import stuff

Reply:
You can reduce the circular imports by just having the conversion code be functional in its own file and not be attached to classes